content/guides/models/track/limits.md

```diff
@@ -164,10 +166,18 @@ If you find you have too many sections and performance is slow, consider the wor
 
 {{< img src="/images/track/section_prefix_toggle.gif" alt="Toggling section creation" >}}
 
+### Metric count
+
+When logging 5000 - 100,000+ metrics per run, W&B recommends using (Manual workspace mode)[https://docs.wandb.ai/guides/app/features/panels/#workspace-modes].  This allows you to rapidly add plots for exactly the metrics you need to examine, without the performance overhead of all the other metrics (which are stored as usual, but not automatically plotted).  In Manual mode, you can easily add and remove panels in bulk as you choose to explore different sets of metrics-   To enter manual mode, first hit Reset workspace in the top workspace menu.  No run or metric data is lost when you reset your workspace--it is purely about the display.
+
 ### File count
 
 Keep the total number of files uploaded for a single run under 1,000. You can use W&B Artifacts when you need to log a large number of files. Exceeding 1,000 files in a single run can slow down your run pages.
 
+### Reports vs. Workspaces
+
+Reports allow for free-form composition of arbitrary arrangements of panel types, text, and media, making them powerful tools for sharing your insights with colleagues.  However, for high-density analysis of dozens-to-thousands of metrics across hundreds-to-hundreds-of-thousands of runs, Workspaces provide a far more performant experience--their structure allows for more optimized caching, querying, and loading behaviors. For single-project use cases centered on analysis and not presentation, W&B recommends using workspaces when you have arrays of 20 or more plots. 
```

Comment:
```suggestion
A report is a free-form composition of arbitrary arrangements of panels, text, and media, allowing you to easily share your insights with colleagues.

By contrast, a workspace allows high-density and performant analysis of dozens to thousands of metrics across hundreds to hundreds of thousands of runs. Workspaces have optimized caching, querying, and loading capabilities, when compared to reports. Workspaces are recommended for a project that is used primarily for analysis, rather than presentation, or when you need to show 20 or more plots together.
```